docs/release-notes/v0.41.0.md

```diff
@@ -0,0 +1,81 @@
+## Announcing Radius v0.41.0
+
+Today we're happy to announce the release of Radius v0.41.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
```

Comment:
```suggestion
## Highlights

### Improved UI for `rad init` CLI prompt in a small terminal window
We have resolved a usability issue where the text in the `rad init` CLI user interface gets cropped out of the visible area in a small terminal window. The text display area for the `rad init` UI now gets dynamically changed based on the terminal window dimensions, allowing for the text to be displayed instead of being cropped out of the visible display area.
```